config/projects/datadog-agent.rb

```diff
@@ -1,5 +1,5 @@
 name 'datadog-agent'
-maintainer 'Datadog Packages <package@datadoghq.com>'
+maintainer 'Etienne Omnitests <etienne.lafarge@datadoghq.com>'
```

Comment:
This gets populated into every package we deploy and will show up in things like apt-cache info - are you sure you should change this?